examples/modular-drivers/Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
-name = "micro-rdk-esp32-sensor-examples"
-resolver = "2"
-version = "0.2.0"
-edition = "2021"
-authors = ["Viam, inc. <contact@viam.com>"]
-resolver = "2"
-rust-version = "1.75"
-license = "AGPL-3.0"
+name = "modular-drivers"
+authors.workspace = true
+description.workspace = true
+edition.workspace = true
+license.workspace = true
```

Comment:
Interestingly, the example modules do *not* have the same license as the overall workspace.

The micro-rdk is mostly AGPL, but the example modules are Apache.

Reply:
I may have been too oblique here: please change this to reflect the actual license of the examples (Apache)